src/common/regions.ts

```diff
@@ -119,16 +141,62 @@ const counties: County[] = chain(state_county_map_dataset)
   })
   .value();
 
+const countiesByFips = fromPairs(
+  counties.map(county => [county.fipsCode, county]),
+);
+
+// Seattle-Tacoma, WA -> seattle-tacoma_wa
```

Comment:
@ghop02 just added our location pages to our normal API endpoints I believe.  What's the status for CBSA location pages in the API?

Should we put these urls into the json file and then share the json file with the backend API code?

Reply:
https://github.com/covid-projections/covid-data-public/blob/master/scripts/update_can_location_urls.py We update them here

Reply:
fwiw I think this would be helpful to have in the json file both for that and so we don't accidentally change them in the future.

Reply:
I will add this to the script so they are hard-coded in the JSON file